src/app/shared/validationMessages.model.ts

```diff
@@ -43,7 +44,10 @@ export const validationPatterns = {
   'versionTag': '^[a-zA-Z0-9]+([-_\.][a-zA-Z0-9]+)*$',
   'reference': '[\\w-]+((/|.)[\\w-]+)*',
   'workflowDescriptorPath': '^\/([^\\\/\?\:\*\|\<\>]+\/)*[^\\\/\?\:\*\|\<\>]+\.(cwl|wdl|yaml|yml)',
-  'workflowName': '[a-zA-Z0-9]+([-_][a-zA-Z0-9]+)*'
+  'workflowName': '[a-zA-Z0-9]+([-_][a-zA-Z0-9]+)*',
+  'cwlTestParameterFilePath': '^/([^\/?:*|<>]+/)*[^\/?:*|<>]+.(json|yml|yaml)$',
```

Comment:
If the pattern is the same, you only need to have one of them.  It doesn't need to match the formErrors